tests/code_size/math_wasm.json

```diff
@@ -3,8 +3,8 @@
   "a.html.gz": 431,
   "a.js": 119,
   "a.js.gz": 132,
-  "a.wasm": 2721,
-  "a.wasm.gz": 1635,
-  "total": 3513,
-  "total_gz": 2198
+  "a.wasm": 2769,
+  "a.wasm.gz": 1662,
+  "total": 3561,
+  "total_gz": 2225
```

Comment:
This increase looks unexpected, I wonder what causes it?

Reply:
This change was invalid.  No regressions to the code_size tests as part of this change.